src/adapter/src/coord.rs

```diff
@@ -2858,6 +2858,37 @@ pub fn serve(
             .expect("missing EpochMillisseconds timestamp")
             .clone();
 
+        // Choose a time at which to boot. This is the time at which we will run
+        // internal migrations.
```

Comment:
> This is the time at which we will run internal migrations.

This used to be true, and I thought it was currently true, but looking through the code it doesn't actually look like this is true anymore.

Reply:
Then we can remove the call to `txn.set_timestamp` from `open.rs`.